modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/HorizontalCardTag.java

```diff
@@ -69,11 +70,12 @@ public String getTitle() {
 		HorizontalCard horizontalCard = getHorizontalCard();
 
 		if ((_title == null) && (horizontalCard != null)) {
-			title = horizontalCard.getTitle();
+			title = HtmlUtil.escape(horizontalCard.getTitle());
```

Comment:
Why the double escape? I think this one should be unnecessary since you're already escaping below, isn't it?

Reply:
You're right. Thanks